### PR TITLE
More fixes for ensure option for FreeBSD services

### PR DESCRIPTION
**Pull request by timp87**
1. Service's rc_script name can differ from rcvar which is used to make service start from boot.
Extract rcvar from rc script.
2. As additional profit we can even complain about incorrect service name.
3. As another profit we can get current rcvar value ($rcvalue) and decide if we actually need to make our manipulations with rc.conf*.
4. Wrap /etc/rc.conf.local editing inside is_file to calm down warnings.
5. Get current service status (already running?) and do our stuff if it's really needed.
6. Edit regexp when we need to enable service in /etc/rc.conf* files. I. e. catch incorrect assignments like 'mysql_enable="bla"'.

perltidy is happy if you care =)

### CODE REVIEW

File: lib/Rex/Service/FreeBSD.pm [hunk 1]
```diff
@@ -42,22 +43,43 @@ sub ensure {
 
   my $what = $options->{ensure};
 
+  my $rccom = "/usr/sbin/service $service rcvar";
+  my $rcout = i_run $rccom;
+  if ( $? != 0 ) {
+    Rex::Logger::info( "Running `$rccom` failed", "error" );
+    return 0;
```
Comment (timp87) [inline lib/Rex/Service/FreeBSD.pm]: we should decide here and there (line 56) if we really have to do here.

Comment (ferki) [inline lib/Rex/Service/FreeBSD.pm]: Looking around in the code, I believe `return 0` is correct here.

**Comment by ferki** [comment]
@timp87: thanks for the updates, this is now merged after rebasing, and squashing!

**Comment by ferki** [comment]
@timp87: I had to run perltidy afterwards, but it is fine. I activated automated tests for it, so we should be able catch untidy code earlier :)

**Pull request closed without merging**